src/main/resources/docs/description/description.json

```diff
@@ -1931,12 +1937,24 @@
     "description": "Flip leading $ and \" if this should be a quoted substitution.",
     "timeToFix": 5
   },
+  {
+    "patternId": "SC2248",
+    "title": "# Warn about variable references without braces.",
```

Comment:
Are these # supposed to be here_

Reply:
Nice catch! 👌It was not supposed, fixed it ✅